src/core/Cheatsheet.ts

```diff
@@ -28,104 +26,76 @@ interface CheatSheetCommandArg {
   outputPath: string;
 }
 
-export default class Cheatsheet {
-  private disposables: vscode.Disposable[] = [];
+export async function showCheatsheet({
```

Comment:
note: we don't bother with a class at all; we just export a couple functions here

also note that this file doesn't include any meaningful changes; just migrate command registration to `extension.ts`